domains/shawn.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "shawnmarck",
+    "email": "shawn.marck@gmail.com"
+  },
+  "records": {
+    "CNAME": "shawnmarck.github.io"
```

its better to use our URL records (to avoid too many redirection)
```suggestion
    "URL": "https://github.com/shawnmarck"
```